notification/app/notification/services/guardian/GuardianNotificationSender.scala

```diff
@@ -117,8 +117,8 @@ class GuardianNotificationSender(
     }
 
     shard(countWithDefault).map { shard =>
-      ShardedNotification(notification, shard, platform)
-      val shardedNotification = ShardedNotification(notification, shard, platform)
+      ShardedNotification(notification, shard, Some(platform))
```

Comment:
I hadnn't noticed  this before. I don't really get the point of this line. What am I missing? We''re calling apply but doing nothing with the returned object

Reply:
Good spot

Reply:
:+1: